academy/5-cosmjs/multi-msg.md

```diff
@@ -279,14 +286,45 @@ const result = await signingClient.signAndBroadcast(
             typeUrl: "/cosmos.staking.v1beta1.MsgDelegate",
             value: {
                 delegatorAddress: alice,
-                validatorAddress: your_selected_validator_address,
+                validatorAddress: validator,
                 amount: { denom: "uatom", amount: "1000", },
             },
           },
     ],
     "auto",
 )
 ```
+Note: If you get the error "Error: Gas price must be set in the client options when auto gas is used." You can fix it by changing the command to the following:
```

Comment:
I would say to change the `"auto"` part to something else and give an example. Here it sounds too much like we have to change everything (scary).

Reply:
removed command and changed auto